templates/tfengine/components/project/bastion_hosts/main.tf

```diff
@@ -0,0 +1,43 @@
+{{- /* Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */ -}}
+
+{{range .bastion_hosts -}}
+module "{{resourceName .name}}" {
+  source = "git::https://github.com/terraform-google-modules/terraform-google-bastion-host.git?ref=umairidris-patch-1"
```

Comment:
Leave a TODO this this should change to an official release, once available.

Reply:
Done.